### ISSUE TITLE
Not getting gifs on Sankaku

### ISSUE THREAD

**Issue by Silkana** [Bionus/imgbrd-grabber#563]
I don't seem to be downloading gifs from this website, only jpgs and pngs.

Nothing to do with the previous issue but is there a mean to bypass Sankaku img limitation?
I'm no expert but it seems to stop downloading images at some point when I line more than 100/200 pictures. Like when you browse it manually with the "Oops, slow down"

**Comment by Bionus** [comment]
I can't reproduce on my build. I just downloaded a few `animated_gif` images and they were all proper `.gif` files. Do you have a md5 that reproduces this issue?

As for the limits, you have to set the throttle settings in Sankaku's options, so that you don't go over their limit. From memory, 3s/page, 1s/image, and 180s/failure should do the trick. It slows the process a bit but at least it completes (nothing more I can do about that).

**Comment by Silkana** [comment]
Thanks for the threshold informations.

I tried again to get gifs. ( let's say with the tag Tinklebell 'cause there's a lot of gifs) and I just get thumbnails of the supposed animated gifs.

I'm probably doing something wrong.

**Comment by Silkana** [comment]
So yeah, the gif problem persists only on Sankaku.

i tried on e621 and boorus and i get the gifs perfectly fine.
do you have any idea what could be the problem?

**Comment by Aelphais** [comment]
I'd like to add that webms and such do not work from Sankaku either. Using version 4.5.3. I just tried a few overwatch webms and got nothing but the thumbnail.

Clicking "More details" on one of the webms produces the following:

ID 5474142
MD5 f5549f82a044a0817240b84358eec1bf

Page: https://chan.sankakucomplex.com/post/show/5474142
URL: http://cs.sankakucomplex.com/data/f5/54/f5549f82a044a0817240b84358eec1bf.jpg
Source: Unknown
Sample http://c.sankakucomplex.com/data/preview/f5/54/f5549f82a044a0817240b84358eec1bf.jpg
Preview: http://c.sankakucomplex.com/data/preview/f5/54/f5549f82a044a0817240b84358eec1bf.jpg


I'm relatively certain that the URL is incorrect. I should be .webm, but isn't. I just upgraded to the latest version a day or two ago from a 4.4.x version and haven't got a webm or gif since.
